com.regnosys.rosetta/src/com/regnosys/rosetta/generator/java/util/JavaNames.xtend

```diff
@@ -81,8 +81,6 @@ class JavaNames {
 			RosettaRecordType:
 				JavaType.create(JavaClassTranslator.toJavaFullType(type.name)) ?:
 					JavaType.create(packages.defaultLibRecords.name + '.' + type.name.toFirstUpper)
-			RosettaExternalFunction:
-				createJavaType(packages.defaultLibFunctions, type.name)
```

Comment:
A `RosettaExternalFunction` used to be a `RosettaType` for some reason. I fixed that.